sabre1041/managing-ocp-install-beyond
# Update credential modules to use tower-cli

The best way to test a full run is to run `aws_lab_launch.yml` using an AMI that has been recently built (with no credentials built in).

To test thoroughly, do the following:

- Specify the new variable `tower_credential_machine_ssh_key_data_file` (this was previously `tower_credential_machine_ssh_key_data`) and should be the local filename and path to your private key either on the CLI via -e or in a secrets file
- Launch `aws_lab_launch.yml` with `tower_config: none`
- Login with your student ID (user-X) and whatever password you set with `tower_password`
- Verify no Machine or Cloud credentials are baked in
- Redpeloy again but this time with `tower_config: self`
- Login and verify credentials are added for Machine and Cloud
- Launch `Self` job to ensure all job and workflow templates are created
- Optionally run `tower_config: full` or `tower_config: test` to verify
